### ISSUE TITLE
History of recent peripheral venous catheter

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10690]
**Preferred term label:**
History of recent peripheral venous catheter

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Sibling term https://hpo.jax.org/browse/term/HP:6000168

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Septic thrombophlebitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001118
- label: History of recent peripheral venous catheter
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: A history of placement of a peripehral venous catheter in the hours, days, or weeks before the current presenting complaint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10690
- synonyms:
